submissions/models.py

```diff
@@ -205,6 +205,25 @@ def create_reset_score(cls, student_item):
             reset=True,
         )
 
+    @classmethod
+    def create_override_score(cls, student_item_model, points_possible, override_grade):
+        """
+        Create an "override" score (a score with a null submission and reset False).
+
+        Args:
+            student_item_model (StudentItem): The student item model.
+            points_possible (string): The max points possible for this submission.
+            o
+
+        """
+
+        return cls.objects.create(
+            student_item=student_item_model,
+            submission=None,
+            points_earned=float(override_grade),
+            points_possible=float(points_possible),
```

Comment:
Ah, I see above that they're originally strings; should we have some form of error handling?